middleware/parse/parse_test.go

```diff
@@ -107,6 +108,8 @@ func TestParseMiddleware(t *testing.T) {
 		})
 		assert.NoError(t, result.Body.Close())
 		assert.Equal(t, http.StatusBadRequest, result.StatusCode)
+		assert.NotNil(t, request.Extra[goyave.ExtraRequestError{}])
+		assert.Contains(t, request.Extra[goyave.ExtraRequestError{}].([]error), fmt.Errorf("request.json-invalid-body"))
```

Comment:
You may face a type assertion panic here

Reply:
You're right, but that would not be a bad thing. This case is testing these lines in code 
```
r.Extra[goyave.ExtraRequestError{}] = []error{
    fmt.Errorf("request.json-invalid-body"),
}
```
which always returns `[]error{}`. If this case ever panics, it probably should.

Reply:
You should test it with NotPanics then

https://pkg.go.dev/github.com/stretchr/testify/assert#NotPanics